### PR TITLE
Fix:memory leak during forcesync

### PR DESCRIPTION
**Pull request by nlrcomcast**
[Field][TCXB7|TCXB8][7.3p3s1] webconfig memory is leaking with uptime

### CODE REVIEW

File: src/webcfg_multipart.c [hunk 1]
```diff
@@ -2010,7 +2010,9 @@ void delete_multipart()
 	{
 		temp = head;
 		head = head->next;
-		WebcfgDebug("Deleted mp node: temp->name_space:%s\n", temp->name_space);
+		WebcfgInfo("Deleted mp node: temp->name_space:%s\n", temp->name_space);
```
Comment (sadhyama) [inline src/webcfg_multipart.c:2013]: Move it to Debug after testing

File: src/webcfg_db.c [hunk 2]
```diff
@@ -767,7 +767,11 @@ void delete_tmp_list()
     {
         temp = head;
 	head = head->next;
-	WebcfgDebug("Delete node--> temp->name %s temp->version %lu temp->status %s temp->isSupplementarySync %d temp->error_details %s temp->error_code %lu temp->trans_id %lu temp->retry_count %d temp->cloud_trans_id %s\n",temp->name, (long)temp->version, temp->status, temp->isSupplementarySync, temp->error_details, (long)temp->error_code, (long)temp->trans_id, temp->retry_count, temp->cloud_trans_id);
+	WebcfgInfo("Delete node--> temp->name %s temp->version %lu temp->status %s temp->isSupplementarySync %d temp->error_details %s temp->error_code %lu temp->trans_id %lu temp->retry_count %d temp->cloud_trans_id %s\n",temp->name, (long)temp->version, temp->status, temp->isSupplementarySync, temp->error_details, (long)temp->error_code, (long)temp->trans_id, temp->retry_count, temp->cloud_trans_id);
```
Comment (sadhyama) [inline src/webcfg_db.c:770]: Move it to Debug after testing

**Review by sadhyama — CHANGES_REQUESTED** [review]

File: src/webcfg_multipart.c (hunk 1 above)
Comment (nlrcomcast) [reply src/webcfg_multipart.c]: done

File: src/webcfg_db.c (hunk 2 above)
Comment (nlrcomcast) [reply src/webcfg_db.c]: done

**Review by sadhyama — APPROVED** [review]

**Pull request merged**